### PR TITLE
Unify the backup path on the validate_encrypt module

### PR DESCRIPTION
**Pull request by b10n1k**
- Related ticket: https://progress.opensuse.org/issues/71728
- [Verification run](https://openqa.suse.de/tests/overview?distri=sle&version=15-SP3&build=b10n1k%2Fos-autoinst-distri-opensuse%2371728_validate_encrypt_powervm)

### CODE REVIEW

**Comment by b10n1k** [comment]
- [ ] VR
- [ ] Some files are the same but i rather keep them as it is because the naming wont make clear in which test belongs

File: lib/validate_encrypt_utils.pm [hunk 1]
```diff
@@ -184,9 +184,6 @@ Where C<%args> expects the following parameters:
 =item C<$backup_file_info>
   unique string to match with the info of the backup file
 
-=item C<$backup_path>
```
Comment (rwx788) [inline lib/validate_encrypt_utils.pm:187]: I don't understand why you want to remove it? Let's either keep it and have it in the test data, or let's remove it completely, so it's hard-coded.

**Review by rwx788 — COMMENTED** [review]
Also, please, follow guidelines for git commit messages: https://chris.beams.io/posts/git-commit/

File: lib/validate_encrypt_utils.pm (hunk 1 above)
Comment (b10n1k) [reply lib/validate_encrypt_utils.pm]: restored

**Review by rwx788 — COMMENTED** [review]
We also are missing change in `schedule/yast/encryption/cryptlvm_iscsi.yaml`, could you please adjust it too?

**Comment by b10n1k** [comment]
- [ ] comment addressed
- [ ] VR done

**Pull request merged**